kfs-core/src/main/java/edu/arizona/kfs/vnd/service/impl/EcustomsServiceImpl.java

```
@@ -0,0 +1,698 @@
+            documentService = SpringContext.getBean(DocumentService.class);
+        }
+        return documentService;
+    }
+
+    private static BusinessObjectService getBusinessObjectService() {
+        if (businessObjectService == null) {
+            businessObjectService = SpringContext.getBean(BusinessObjectService.class);
+        }
+        return businessObjectService;
+    }
+
+    private static LookupService getLookupService() {
+        if (lookupService == null) {
+            lookupService = SpringContext.getBean(LookupService.class);
+        }
+        return lookupService;
+    }
+
+    private static EcustomsFileService getEcustomsFileService() {
+        if (ecustomsFileService == null) {
+            ecustomsFileService = SpringContext.getBean(EcustomsFileService.class);
+        }
+        return ecustomsFileService;
+    }
+
+    @Override
+    public boolean createEcustomsDailyFile(String jobName, Date jobRunDate) throws Exception {
+        LOG.debug("createEcustomsDailyFile(): jobRunDate=" + jobRunDate + ", start-time=" + new Date());
+        boolean processVendorDetails = false;
+        try {
+            boolean deleteOldDailyDone = getEcustomsFileService().deleteOldDailyDoneFiles();
+            LOG.debug("getEcustomsFileService().deleteOldDailyDoneFiles() returned " + deleteOldDailyDone);
+            dataFile = getEcustomsFileService().getDailyBatchDataFile(jobRunDate);
+            vendorCountFile = getEcustomsFileService().getDailyBatchVendorCountFile(jobRunDate);
+            batchName = VendorConstants.ECUSTOMS_BATCH_DAILY;
+            List<VendorDetail> vendorDetails = getDailyVendorDetailList(jobRunDate);
+            processVendorDetails = processVendorDetails(jobRunDate, vendorDetails);
+            getEcustomsFileService().createDailyBatchDoneFile(jobRunDate);
+        } catch (Exception e) {
+            LOG.error("Exception: " + e.getMessage());
+            throw new Exception(e.getMessage());

```

Originally, I had the LOG statement there to set a break point. In the next commit, this has been removed. For the try/catch, just remove it and let the method just throw whatever Exception pops up?